### PR TITLE
Add Docker docs for Auditbeat

### LINKED ISSUE #5132: Document running Auditbeat under Docker
The Auditbeat documentation needs a Docker section like the other Unix Beats have.

### PR DESCRIPTION
**Pull request by jarpy**
Closes #5132

### CODE REVIEW

**Comment by ruflin** [comment]
@jarpy Could you rebase this on the most recent version of master? This should also get the Jenkins build green.

**Comment by jarpy** [comment]
Of course. Done.

(Side-effect of forgetting to raise the PR sooner).

File: auditbeat/docs/running-on-docker.asciidoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+["source","sh",subs="attributes"]
+----
+docker run --cap-add=AUDIT_CONTROL,AUDIT_READ {dockerimage}
+----
+
+It is also essential to run {beatname_uc} in the host PID namespace.
+
+["source","sh",subs="attributes"]
+----
+docker run --pid=host {dockerimage}
```
Comment (andrewkroh) [inline auditbeat/docs/running-on-docker.asciidoc]: I expect people will copy/paste this so I recommend showing both the `-cap-add` and `--pid=host` on the same line.

Comment (jarpy) [reply auditbeat/docs/running-on-docker.asciidoc]: Good point, I was trying to be minimal to draw attention to the current topic, but you're probably right about cut-n-paste.

Comment (jarpy) [reply auditbeat/docs/running-on-docker.asciidoc]: Fixed in 29f4c6fe3ee

**Pull request merged**